stan/math/prim/scal/meta/operands_and_partials.hpp

```diff
@@ -28,10 +28,11 @@ namespace internal {
 template <typename ViewElt, typename Op>
 class ops_partials_edge {
  public:
-  empty_broadcast_array<ViewElt, Op> partials_;
+  typedef empty_broadcast_array<ViewElt, Op> partials_t;
+  partials_t partials_;
```

Comment:
Oops, I think we can actually revert the changes on this file - we don't need the `typedef` anymore and we typically add a commented out argument name to make it clear that we don't use the argument (the `/* op */` below)